src/app/components/BibPage/BibDetails.jsx

```diff
@@ -52,7 +54,7 @@ class BibDetails extends React.Component {
   }
 
   /*
-   * getDefinitionObject(bibValues, fieldValue, fieldLinkable)
+   * getDefinitionObject(bibValues, fieldValue, fieldLinkable, fieldSelfLinkable, fieldLabel)
```

Comment:
Line break at fieldSelfLinkable... a la line 152/153?

Reply:
Ooh so we have an eslint rule to keep lines under 100 characters in length. Line 152/153 had to be broken into two to be under 100 chars. This one on line 57 is fine.

Reply:
Just trying to keep you honest. :) -- I didn't update and check against my ESLinter.

Reply:
Ha, thanks.